rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -13,6 +13,18 @@ toc::[]
 [id="rosa-new-changes-and-updates_{context}"]
 == New changes and updates
 
+[id="rosa-q2-2024_{context}"]
+=== Q2 2024
+
+* **ROSA CLI update.** The ROSA CLI (`rosa`) was updated to a new version. For information about what has changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.37[ROSA CLI release notes]. For more information about the ROSA CLI (`rosa`), see xref:../cli_reference/rosa_cli/rosa-get-started-cli.adoc#rosa-about_rosa-getting-started-cli[About the ROSA CLI].
+
+* **Enabled additional custom security groups for {hcp-title}.** Starting with ROSA CLI version 1.2.37, you can now use the `--additional-security-group-ids <sec_group_id>` when creating machine pools on {hcp-title} clusters by using the ROSA CLI. For more information, see xref:../rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.html#creating_machine_pools_cli_rosa-managing-worker-nodes[Creating a machine pool using the ROSA CLI] and the xref:../cli_reference/rosa_cli/rosa-manage-objects-cli.html#rosa-create-machinepool_rosa-managing-objects-cli[create machinepool] section of the ROSA CLI reference.
++
+[IMPORTANT]
```

Comment:
Why is this section in the What's New / Release Notes? Remove this from here and consider adding only in the feature section.